brands/shop/convenience.json

```diff
@@ -1294,17 +1294,25 @@
   },
   "shop/convenience|Tesco": {
     "count": 53,
-    "nomatch": ["amenity/fuel|Tesco"],
+    "nomatch": [
+      "amenity/fuel|Tesco",
+      "shop/convenience|Tesco Express"
+    ],
     "tags": {
       "brand": "Tesco",
+      "brand:wikidata": "Q487494",
+      "brand:wikipedia": "en:Tesco",
       "name": "Tesco",
       "shop": "convenience"
     }
   },
   "shop/convenience|Tesco Express": {
     "count": 728,
+    "nomatch": ["shop/convenience|Tesco Express"],
```

Comment:
is it wrong name or something else went wrong? Because nomatch to `shop/convenience|Tesco Express` in `shop/convenience|Tesco Express` is not necessary.

Reply:
That was supposed to be nomatch `amenity/fuel|Tesco`, I copied the block from Tesco and deleted the wrong one. I've added a new commit which fixes this.